js/components/ReportSummary.vue

```diff
@@ -23,7 +23,8 @@
           <a class="delete" v-if="canDelete" @click="deleteSummary">Delete <i class="far fa-trash-alt"></i></a>
         </div>
         <div v-if="editing && canEdit" class="edit-form container">
-          <ReportSummaryForm :hideFormTitle=true
+          <ReportSummaryForm :key="model.id"
```

Comment:
What's the reason for the use of `:key` here?

Reply:
Initially I thought there needed to be a key since there were multiple forms on the page, but maybe that's only for [v-for](https://vuejs.org/v2/guide/list.html#key)? I may have misunderstood its usage. I think it can be removed.